client/src/common/client.ts

```diff
@@ -2,6 +2,8 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
+/// <reference path="../../typings/vscode.proposed.codeActionAI.d.ts" />
```

Comment:
Please add this to both convertes and not to the client. It is IMO not necessary here: